examples/inspection/plot_individual_conditional_expectation.py

```diff
@@ -0,0 +1,162 @@
+"""
+==============================================
```

Comment:
I think that I would not introduce a new example but instead modify the existing PDP example. We could add a section on ICE to explain what is the intention when plotting the individual response and add a little paragraph when discussing the plot to show the usefulness of ICE in regards to the PDPs.